examples/stackoverflow/foo.sh

```
@@ -0,0 +1,3 @@
+if ! grep -q publish_to "pubspec.yaml"; then
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)